repository-data/webfiles/src/main/resources/site/freemarker/nationalindicatorlibrary/nihub.ftl

```diff
@@ -9,7 +9,7 @@
     </#if>
 </section>
 
-<section id="popular-topics" class="document-content">
+<section id="popular-topics" class="document-content" title="popular-topics">
```

Comment:
It seems that the `title` attribute is used here solely to have something that the acceptance test can find the section element by. Without actually displaying the page to test, I think it has an unintended side effect of the title being displayed as a tool-tip whenever mouse cursor hovers over within the boundaries of the section element.

Could we have it changed to use `data-uipath`, which was introduced precisely as a hidden element used to target DOM elements from within acceptance tests?